### ISSUE TITLE
Gmail shortcuts request

### ISSUE THREAD

**Issue by seisvelas** [FlowCrypt/flowcrypt-browser#3629]
>I use a lot of the gmail keyboard shortcuts (like 'r' for reply). It would be nice if you offered some sort of similar thing (either overriding gmail's shortcuts or adding a new one).

ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwLtsvPJlpBVzkVgkbQdJsBhqbf

Is this desirable for us? Just bringing it up in case this is something we would ever want to pursue, since we received the request.

**Comment by limonte** [comment]
Hey @encody and thank you for this request!

Most of shortcuts are already there, but we're still missing the docs. There's a task for me in another repo to create that page: https://github.com/FlowCrypt/flowcrypt-web/issues/262

Could you please provide list all the shortcuts that you're missing and then @tomholub will decide what to do next.

**Comment by limonte** [comment]
Thank you @encody for the list of expected shortcuts.

@tomholub I would suggest to prioritize this issue because it's at the same time feature-request and bug-fix.

Here's the bug which is happening in Firefox only:



https://user-images.githubusercontent.com/6059356/120043053-eee79c80-c013-11eb-97ee-bdd7d8d6dd77.mp4




### Steps to reproduce:

1. Open any encrypted message
2. Press `r` two times

### Expected behavior

Encrypted reply with "r" in it.

### Actual behavior

Invisible unencrypted reply with "r" in it.

---

Proposed action steps:

1. In scope of this issue add support for `r` and `a` shourtcuts.
2. Create another issue if needed for "Secure Compose" action. Let's duscuss the shortcut, I don't think we should override Gmail's `c` shortcut.
3. Create another issue if needed for `?` shortcut.

**Comment by tomholub** [comment]
thank you for looking into this - sounds good

**Comment by limonte** [comment]
Blocked by #3789
